joblib/test/test_func_inspect.py

```diff
@@ -193,11 +188,22 @@ def test_bound_methods():
     assert filter_args(a.f, [], (1, )) != filter_args(b.f, [], (1, ))
 
 
-def test_filter_args_error_msg():
-    """ Make sure that filter_args returns decent error messages, for the
-        sake of the user.
+@parametrize(['exception', 'regex', 'func', 'args'],
+             [(ValueError, 'ignore_lst must be a list of parameters to ignore',
+               f, ['bar', (None, )]),
+              (ValueError, 'Ignore list: argument \'(.*)\' is not defined',
+               g, [['bar'], (None, )]),
+              (ValueError, 'Wrong number of arguments',
+               h, [[]])
+              ])
+def test_filter_args_error_msg(exception, regex, func, args):
+    """
+    Make sure that filter_args returns decent error messages, for the sake
```

Comment:
Was this docstring too long? It it was try to follow [PEP0258](https://www.python.org/dev/peps/pep-0258/). Short description one the first line and then a blank line and then long description.